modes/from-source/docker-compose.yml

```diff
@@ -309,7 +309,7 @@ services:
       - internal
     ports:
       - 9000:9000
-      - 9090:9090
+      - 9090
```

Comment:
This will lead to a random host port allocation. While it previously was a mapping from container port 9090 to host port 9090 it is now a mapping from container port 9090 to a random host port. Is this the intended effect?

Reply:
In order to connect from the host you'd need to run `docker compose port minio 9090` to find out where it was mapped to.

Reply:
9090 seems to be the management dashboard. We usually don't use this so it might be actually o.k.

Reply:
i believe that this is fine, since only 9000 is being accessed from the outside.
As i understood it, internally its using 9090 so the actual 'host port' is not needed.
I have tested this locally and it seems to work just fine.